rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -54,7 +54,127 @@ paths:
+      operationId: poll
+      tags:
+        - transform
+      responses:
+        '200':
+          description: The transform is still in progress.
+          headers:
+            Content-Version:
+              $ref: '#/components/headers/ContentVersion'
+        '201':
+          description: Transform complete. Returning metadata locations.
+          headers:
+            Content-Version:
+              $ref: '#/components/headers/ContentVersion'
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    format:
+                      $ref: '#/components/schemas/MetadataFormat'
+                    location:
+                      description: The URI for where the metadata is located.
+                      type: string
+                      format: uri
+                      minLength: 1
+                      maxLength: 2048
+                      example: 'https://www.example.com/mis/30f14c6c1fc85cba12bfd093aa8f90e3'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          description: The transformation failed.
+          headers:
+            Content-Version:
+              $ref: '#/components/headers/ContentVersion'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '501':
+          $ref: '#/components/responses/NotImplemented'
+        'default':
+          $ref: '#/components/responses/DefaultError'
+    delete:
+      summary: Delete the transformation request
+      description: >
+        After retrieving the output of the transformation, the client should call this endpoint to remove the request.
+      operationId: delete
+      tags:
+        - transform
+      responses:
+        '204':
+          description: The transformation request was successfully deleted.
+          headers:
+            Content-Version:
+              $ref: '#/components/headers/ContentVersion'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '501':
+          $ref: '#/components/responses/NotImplemented'
+        'default':
+          $ref: '#/components/responses/DefaultError'
+  /transform/{TransformId}/{MetadataFormat}:
+    parameters:
+      - $ref: '#/components/parameters/TransformId'
+      - name: MetadataFormat
```

Comment:
❓ Why do you inline MetadataFormat, but use a reference for TransformId?

Reply:
Because the `MetadataFormat` (now called `MetadataType` in response to other comments) is only used as a path parameter in this one place. `TransformId` is a path parameters used in multiple places.